build.sbt

```diff
@@ -81,7 +81,7 @@ lazy val semanticdb3 = crossProject
     ignoreMimaSettings,
     protobufSettings,
     PB.protoSources.in(Compile) := Seq(file("semanticdb/semanticdb3")),
-    version := "3.1.0"
+    version := "3.2.0"
```

Comment:
I suggest we don't update the version number in PRs like here and let sbt-dynver set it automatically instead from the git tag.

Reply:
The fact that the versions of `semanticdb/` modules and `scalameta/` modules are the same is an accident. I think that if we change the parser that should bump the parser version, but not the semanticdb version.

Reply:
I agree in theory, but keeping separate versions in the same repo complicates release a lot. At least sbt doesn't support it nicely. Scalameta depends on semanticdb so I think it makes sense to release them in tandem.

Reply:
Okay. Later on I'd like to move `semanticdb/` into a separate repository, but for now I just don't have time for that.

Reply:
sure we can move it to a separate repo later down the road.